definitions/pipelines/Chipseq_alignment.cwl

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "Chipseq alignment with qc and creating homer tag directory"
+requirements:
+    - class: SchemaDefRequirement
+      types:
+          - $import: ../types/labelled_file.yml
+    - class: SubworkflowFeatureRequirement
+inputs:
+    reference: string
+    bams:
+        type: File[]
+    readgroups:
+        type: string[]
+    mills:
+        type: File
+        secondaryFiles: [.tbi]
+    known_indels:
+        type: File
+        secondaryFiles: [.tbi]
+    dbsnp_vcf:
+        type: File
+        secondaryFiles: [.tbi]
+    omni_vcf:
+        type: File
+        secondaryFiles: [.tbi]
+    intervals:
+        type: File
+    per_base_intervals:
+        type: ../types/labelled_file.yml#labelled_file[]
+    per_target_intervals:
+        type: ../types/labelled_file.yml#labelled_file[]
+    summary_intervals:
+        type: ../types/labelled_file.yml#labelled_file[]
```

Comment:
What did we decide about these intervals? Is there a straightforward way to "blank" them so that we don't report target/base qc metrics? (since there's really no need for them in this pipeline?)